samples/extensions/portability/README.md

```diff
@@ -0,0 +1,83 @@
+<!--
+- Copyright (c) 2022, Holochip
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Vulkan Portability Extension
+
+## Overview
+
+This tutorial, along with the accompanying example code, demonstrates the use of the [VK_KHR_portability_subset](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VK_KHR_portability_subset) extension. 
+When the VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR is set in the ```Instance``` class, Vulkan will consider 
+devices that aren't fully conformant such as [MoltenVk](https://github.com/KhronosGroup/MoltenVK) to be identified 
+as a conformant implementation.  When this happens, use the VkPhysicalDevicePortabilitySubsetPropertiesKHR extension 
+with the [vkGetPhysicalDeviceFeatures2](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#vkGetPhysicalDeviceFeatures2)
+as detailed below to get the list of supported/unsupported features.
+
+This tutorial along with the accompanying code also demonstrates the use of the [VkPhysicalDevicePortabilitySubsetPropertiesKHR]
+(https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.
```

Comment:
It seems the space/line break here after the `vkspec.` breaks the link in markdown.